assets/css/sponsori.css

```diff
@@ -0,0 +1,125 @@
+body{
+    margin: 0;
+}
+
+:root {
+    --black: white;
+    --white: black;
+    --secondary: rgb(237, 28, 36);
+    --secondary-transparent: rgba(237, 28, 35);
+  }
+
+h1{
```

Comment:
Let's avoid writing css on HTML tags. If we import this CSS in another file, it may break the page. Let's change this to a class and apply it to all the titles from the page